Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -1607,6 +1608,15 @@ private void AssignPlayerIndex()
             }
         }
 
+        #if UNITY_EDITOR
```

Comment:
`&& UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS`

Reply:
Ah yes, thanks. I'm always forgetting about the project-wide actions define